HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,88 @@
+# Trace Context HTTP Header Format
```

Comment:
Which systems are committed to support this? Which systems would you like to support this? Might be helpful to `@`-mention people from the latter so we can have any debates before this is merged.